packages/core/src/components/spinner/_spinner.scss

```diff
@@ -211,7 +216,11 @@ $resources-path: "./resources" !default;
   }
 }
 
-@keyframes pt-spinner-animation {
-  from { transform: rotate(0deg); }
-  to   { transform: rotate(360deg); }
+.pt-input-group .pt-spinner {
```

Comment:
i did this cuz it references spinner variables that are not in a common file so i can't import them inside `_input-group.scss`.